srcpkgs/python-pywatchman/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'python-pywatchman'
+pkgname=python-pywatchman
+version=1.4.1
+revision=1
+noarch=yes
+wrksrc="watchman-4.9.0"
+build_wrksrc="python"
+build_style=python-module
+pycompile_module="pywatchman"
+hostmakedepends="python-devel python3-devel python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel"
+depends="python watchman"
+short_desc="Connect and query Watchman to discover file changes (Python 2)"
+maintainer="Hoang Nguyen <hoang@wetrust.io>"
+license="BSD-3-Clause"
+homepage="https://github.com/facebook/watchman/tree/master/python"
+distfiles="https://github.com/facebook/watchman/archive/v4.9.0.tar.gz"
+checksum=1f6402dc70b1d056fffc3748f2fdcecff730d8843bb6936de395b3443ce05322
+alternatives="watchman-make:watchman-make:/usr/bin/watchman-make2 watchman-wait:watchman-wait:/usr/bin/watchman-wait2"
+
+post_install() {
+	vlicense LICENSE
+}
+
+python3-pywatchman_package() {
+	short_desc="${short_desc/2/3}"
+	noarch=yes
+	pycompile_module="pywatchman"
+	hostmakedepends="python3-devel python3-setuptools"
```

Comment:
no need to replicate hostmake and make depends here

Reply:
ack, will fix it tonight